web/landing/gestionnaire/LandingGestionnaire.js

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from "@mui/styles";
+import React from "react";
+import { Header } from "../../common/Header";
+import { Footer } from "../footer";
+import { IntroGestionnaire } from "./sections/IntroGestionnaire";
+import { OutroGestionnaire } from "./sections/OutroGestionnaire";
+import { SetupMobilic } from "./sections/setup/SetupMobilic";
+import { Testimony } from "./sections/Testimony";
+import { WhatsMobilic } from "./sections/whatsmobilic/WhatsMobilic";
+
+import { LandingSection, LandingSectionList } from "../sections/LandingSection";
+
```

Comment:
🟢 Alors c'est quelque chose dont on a parlé pendant la formation accesibiltié, mais ce serait bien de mettre des titres spécifiques pour les pages pour qu'ils apparaissent dans les onglets, ici ce serait "Gestionnaire - Mobilic", j'ai trouvé [cette doc react ça a pas l'air trop compliqué](https://create-react-app.dev/docs/title-and-meta-tags/)